package/MDAnalysis/guesser/base.py

```diff
@@ -0,0 +1,48 @@
+class GuesserBase:
+    context = "base"
+    _guess = {}
+
+    def is_guessed(self, to_guess):
+        """check that the passed atrributes in the to_guess
+        list can be guessed by the class
+
+        Parameters
+        ----------
+        to_guess: list of atrributes to be guessed then added to the universe
+        Returns
+        -------
+        True or False
```

Comment:
This does not ever return False -- it raises a ValueError instead. I think just returning a bool here might work better.